checkout/update-level-cost-with-tax.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * This code recipe automatically calculates the tax and adds
+ * it to the level cost text on the checkout page as soon as a 
+ * member selects their billing country and state. 
+ *
+ * title: Update the Level Cost Text Based on Country and State
+ * collection: checkout
+ * category: tax
+ *
+ * You can add this recipe to your site by creating a custom plugin
+ * or using the Code Snippets plugin available for free in the WordPress repository.
+ * Read this companion article for step-by-step directions on either method.
+ * https://www.paidmembershipspro.com/create-a-plugin-for-pmpro-customizations/
+ */
+
+ function mypmpro_tax_update_script(){
+
+	global $pmpro_levels;
+
+	$level = ( isset( $_REQUEST['level'] ) ? $pmpro_levels[$_REQUEST['level']] : '' );
```

Comment:
This needs to support ?pmpro_level, we can use pmpro_getLevelAtCheckout() instead of getting it like this from the global.